resilience4j-framework-common/src/main/java/io/github/resilience4j/common/retry/configuration/CommonRetryConfigurationProperties.java

```diff
@@ -489,6 +496,10 @@ public Double getRandomizedWaitFactor() {
         }
 
         public InstanceProperties setRandomizedWaitFactor(Double randomizedWaitFactor) {
+            if (randomizedWaitFactor < 0 || randomizedWaitFactor >= 1) {
```

Comment:
Reason for this PR :) done during handling Retry [here](https://github.com/resilience4j/resilience4j/blob/b81dd07aaa10a8f69fa983ed03e14f062048c552/resilience4j-core/src/main/java/io/github/resilience4j/core/IntervalFunction.java#L254)